### ISSUE TITLE
Adjust stock for new imported products

### ISSUE THREAD

**Issue by IMiGS** [openboxes/openboxes#237]
In inventory screen there is an option to adjust stock for multiple items but it doesn't work for items which stock qty at that moment is null. 
It would be great to adjust stock for newly imported products, for example, i have a warehouse with hundreds items, I can easily import them into the system and also bulk edit some positions but now I need to edit the stock position, if I could use "adjust stock" feature (in inventory screen) for every selected item it would save me a lot of time because for the moment i need to go to every separate product and click "record stock" button and then edit stock. 
The implementation of this feature could be that there is  a screen like it is now when you adjust stock for already stocked items but to add a possibility to  also add/change the serial/Lot number.)

**Comment by jmiranda** [comment]
Hmm, interesting. I believe the only thing preventing that from working is that we filter on items with quantity == 0. So that could be a fairly easy change. However, you'd end up seeing a lot of items if you had a long stock history for a particular product. Meaning, any lot number that has ever been received into the system would show up on that page. 

Alternatively, you can use the Import Inventory feature to quickly import inventory items.

**Comment by IMiGS** [comment]
Hm, but for the first import it would have such large stock history.

I tried to use import inventory feature but i got error when tried to import the same file which I used when imported products into the system.
I have send also bug report but the error which i I get is this:

> Error 500: 
> Servlet: 
> URI: 
> Exception Message: Invalid header signature; read 0x20746375646F7250, expected 0xE11AB1A1E011CFD0 
> Caused by: Invalid header signature; read 0x20746375646F7250, expected 0xE11AB1A1E011CFD0 
> Class: GrailsFragmentCachingFilter 
> At Line: [66]

**Comment by jmiranda** [comment]
> Hm, but for the first import it would have such large stock history.

Can you clarify what you mean by that? What's your actual concern?

> I tried to use import inventory feature but i got error when tried to import the same file which I used when imported products into the system.

Can you send the file you used to import to support@openboxes.com? I'll take a look.

**Comment by IMiGS** [comment]
Oh, grammar mistake. I meant, for the first import there would not be such a large stock history but this probably is very individual, because from our view, at the moment. we have products with one serial/lot number and we could easily edit stock with this feature.
But if inventory import gives similar possibility we could go with that. I will send you this file.

**Comment by jmiranda** [comment]
Yes, you should be able to import all of your items (product, lot number, bin location) in a single import. 

And as I noted on my email response, you need to use XLS (Excel) for the inventory import. Each import feature uses either CSV or XLS, but none of them support both at this time. I'll try to fix that when I have time but for now just save your CSV file in XLS format and it should work.
